crates/tools/src/create_creator_account.rs

```diff
@@ -21,11 +22,89 @@ use crate::{
 };
 use gw_types::{bytes::Bytes as GwBytes, prelude::Pack as GwPack};
 
+/// create ETH Address Registry account
+fn create_eth_addr_reg_account(
+    godwoken_rpc_client: &mut GodwokenRpcClient,
+    privkey: &ckb_fixed_hash::H256,
+    fee_amount: u64,
+    config: &Config,
+    scripts_deploy_result: &ScriptsDeploymentResult,
+) -> Result<u32> {
+    let rollup_type_hash = &config.genesis.rollup_type_hash;
+    let from_address =
+        privkey_to_short_script_hash(privkey, rollup_type_hash, scripts_deploy_result)?;
+    let from_id = short_script_hash_to_account_id(godwoken_rpc_client, &from_address)?;
+    let from_id = from_id.expect("Account id of provided privkey not found!");
+
+    let eth_addr_reg_validator_script_hash = {
+        let mut backends = config.backends.iter();
+        let eth_addr_reg_backend = backends
+            .find(|backend| backend.backend_type == BackendType::EthAddrReg)
+            .ok_or_else(|| anyhow!("EthAddrReg backend not found in config"))?;
+        &eth_addr_reg_backend.validator_script_type_hash
+    };
+    let eth_addr_reg_script = Script::new_builder()
+        .code_hash(eth_addr_reg_validator_script_hash.pack())
+        .hash_type(ScriptHashType::Type.into())
```

Comment:
Suggest set args to rollup type hash, same as meta script.
https://github.com/nervosnetwork/godwoken/blob/38207e6852515de40a7e9ad904d480a18e3743be/crates/generator/src/genesis.rs#L75

Reply:
https://github.com/nervosnetwork/godwoken/pull/587/commits/4669cb02ca1732aab5bcc77f7a1ef92896ef74ab